src/gui/widgets/entry_list.py

```
@@ -186,40 +236,81 @@ async def process_entries(self, entries: list[PasswordEntry]):
+                vault_salt = None
+                
+                # Get from API client if available
+                if hasattr(self, 'api_client') and self.api_client:
+                    master_password = self.api_client._master_password
+                    if hasattr(self.api_client, 'user_session') and self.api_client.user_session:
+                        vault_salt = self.api_client.user_session.vault_salt
+                        print(f"Got vault_salt from api_client.user_session: {bool(vault_salt)}")
+                
+                # Try unlocking if we have both
+                if master_password and vault_salt:
+                    print(f"Attempting to unlock vault with retrieved credentials")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)